### PR TITLE
add create form code example to missing languages

### PR DESCRIPTION
**Pull request by jchen293**
Add create form code example to missing languages

### CODE REVIEW

File: official/docs/curl/shipments/create-form.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+curl -X POST https://api.easypost.com/v2/shipments/shp_.../forms \
+  -u "$EASYPOST_API_KEY": \
+  -H 'Content-Type: application/json' \
+  -d '{
+    "form": {
+        // `type` always required
```
Comment (nwithan8) [inline official/docs/curl/shipments/create-form.sh:6]: In shell, comments need to be `# comment`

**Review by nwithan8 — CHANGES_REQUESTED** [review]
Is this missing the C#, Python, Java and Ruby examples as well?

**Comment by jchen293** [comment]
> Is this missing the C#, Python, Java and Ruby examples as well?

Looks like Ryan already added the code example in those four languages.

**Comment by nwithan8** [comment]
> > Is this missing the C#, Python, Java and Ruby examples as well?
> 
> Looks like Ryan already added the code example in those four languages.

So he did, my bad, was looking in the wrong directory.

**Review by nwithan8 — APPROVED** [review]

File: official/docs/curl/shipments/create-form.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
+curl -X POST https://api.easypost.com/v2/shipments/shp_.../forms \
+  -u "$EASYPOST_API_KEY": \
+  -H 'Content-Type: application/json' \
+  -d '{
+    "form": {
+        # type always required
+        # all other elements differ based on form type
```
Comment (Justintime50) [inline official/docs/curl/shipments/create-form.sh:7]: Issue: You cannot have comments in the middle of curl examples. Please move this to line 1

File: official/docs/php/shipments/CreateForm.php [hunk 3]
```diff
@@ -0,0 +1,22 @@
+require_once("path/to/lib/easypost.php");
```
Comment (Justintime50) [inline official/docs/php/shipments/CreateForm.php:1]: Issue: All PHP files must start with `<?php` to render and run properly.

**Review by Justintime50 — CHANGES_REQUESTED** [review]

File: official/docs/curl/shipments/create-form.sh (hunk 2 above)
Comment (jchen293) [reply official/docs/curl/shipments/create-form.sh]: resolved

File: official/docs/php/shipments/CreateForm.php (hunk 3 above)
Comment (jchen293) [reply official/docs/php/shipments/CreateForm.php:2]: resolved

File: official/docs/php/shipments/CreateForm.php [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<?php
+require_once("path/to/lib/easypost.php");
```
Comment (Justintime50) [inline official/docs/php/shipments/CreateForm.php:2]: Please remove this line (we aren't going to include these anymore, see PHP examples PR for an example.)

Comment (jchen293) [reply official/docs/php/shipments/CreateForm.php]: resolved

File: official/docs/curl/shipments/create-form.sh [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+  -H 'Content-Type: application/json' \
+  -d '{
+    "form": {
+        "type": "return_packing_slip",
+        "barcode": "RMA12345678900",
+        "line_items": [
+        {
+            "product": {
+            "title": "Square Reader",
+            "barcode": "855658003251"
```
[the remark this answers is not in the document]
Comment (jchen293) [reply official/docs/curl/shipments/create-form.sh]: resolved

**Review by Justintime50 — COMMENTED** [review]
Sorry, last item - we need print statements to match all the other examples for Go and PHP

**Review by Justintime50 — APPROVED** [review]

**Pull request merged**